### ISSUE TITLE
Required data format for some form fields is not included in the label

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3094]
**HC Assessment Label: Serious**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest
URL: https://forms-staging.cdssandbox.xyz/en/id/134
User agent:  Chrome (v119.0.6045.200)
Language: English and French

Issue Description: 
The expected data format/example for the following form fields are not provided as part of the label of the form controls.

- Telephone number
- E-mail address

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Visual scan of the content

Expected result: The expected data format/example for those form fields should be provided as part of the label text.

**Comment by srtalbot** [comment]
Fixed, now form builders can set the auto-complete value in the user interface. We've also added specific question components like "email" or "name" that sets it automatically.
